infra/tf/configs/metrics.yml

```diff
@@ -0,0 +1,25 @@
+#cloud-config
```

Comment:
Env variables can go in terraform, also are we installing docker inside a docker image?